### ISSUE TITLE
Gen 6 SR Bot Unable to read 3ds ram

### ISSUE THREAD

**Issue by JakeK0202** [drgoku282/PKMN-NTR#150]
Just putting up a separate issue for the SR bot in gen 6. Error saying an error occurred while reading 3ds ram. Here's the log files again.

https://github.com/drgoku282/PKMN-NTR/files/1778791/StartBot_pkmn-ntr.txt

https://github.com/drgoku282/PKMN-NTR/files/1778792/ResumeBot_pkmn-ntr.txt

**Comment by drgoku282** [comment]
For the start mode, try doing a manual soft-reset while connected, disconnect and try to connect again and see if the error stays.

About the resume, are you sure you're starting in the right window? Can somebody confirm? The program expects a certain value in a certain memory address but your game returns other value, that's why the bot detects an error.

**Comment by JakeK0202** [comment]
Yeah I'm still getting the same error when I try it with the manual reset first. As far as resuming the bot, I've tried from several different screens and haven't had any luck with it

**Comment by drgoku282** [comment]
Manual reset not working means there's an error from your PC or from the game. Do you have any other plugin active (NTR, Rosalina, IPS patches, etc.)? Try deactivating them and if you can, try from another PC.

Also, starting from another screen will never work, because the bot expects certain values in the memory that should only happen when you're at said screen.

**Comment by JakeK0202** [comment]
Pretty certain I don't have any plugins active but I will double check later tonight and try with another pic when I can
